### ISSUE TITLE
Do not install test modules

### ISSUE THREAD

**Issue by moubctez** [ncclient/ncclient#415]
*setup.py* should not install `test` modules; here is a patch that fixes that:

```
--- setup.py.orig	2020-08-04 17:57:22.000000000 +0000
+++ setup.py
@@ -49,7 +49,7 @@ setup(name='ncclient',
       author=__author__,
       author_email=__author_email__,
       url="https://github.com/ncclient/ncclient",
-      packages=find_packages('.'),
+      packages=find_packages(exclude=['test', 'test.*']),
       install_requires=install_reqs,
       tests_require=test_reqs,
       license=__licence__,
```

**Comment by fredgan** [comment]
You can submit a PR directly. If needed I can help you to do it.

**Comment by fredgan** [comment]
Resolved by #419, so it should be closed
